Tasks/PipAuthenticateV0/task.json

```diff
@@ -63,7 +63,7 @@
             "endpointId": "tfs:feed",
             "endpointUrl": "{{endpoint.url}}/_apis/packaging/feeds",
             "resultSelector": "jsonpath:$.value[*]",
-            "resultTemplate": "{ \"Value\" : \"{{{id}}}\", \"DisplayValue\" : \"{{{name}}}\" }"
```

Comment:
Changing this value to name to avoid having ids in Yaml. The python endpoint works well with both name and id. So, not breaking.